Easydict/Feature/Service/Ali/AliService.swift

```diff
@@ -37,13 +25,8 @@ class AliService: QueryService {
         NSLocalizedString("ali_translate", comment: "The name of Ali Translate")
     }
 
-    override func hasPrivateAPIKey() -> Bool {
-        let id = Defaults[.aliAccessKeyId] ?? ""
-        let secret = Defaults[.aliAccessKeySecret] ?? ""
-        return !id.isEmpty && !secret.isEmpty
-    }
-
     override public func supportLanguagesDictionary() -> MMOrderedDictionary<AnyObject, AnyObject> {
+        // swiftlint:disable:next todo
```

Comment:
```suggestion
```

Reply:
@tisfeng How do you think of the `todo` rule? Maybe leaving it is a better choice?

Reply:
I believe it will not trigger a warning after removing this line.